### ISSUE TITLE
Viewshed causes app to crash

### ISSUE THREAD

**Issue by garys-esri** [Esri/squad-leader-android#123]
Zoom out and perform several viewshed analyses. Eventually one of them will make the app crash.

**Comment by jrweakland** [comment]
@garys-esri - Do you think this might be related to https://github.com/Esri/squad-leader-android/issues/119?

**Comment by garys-esri** [comment]
Maybe. The viewshed is still in beta in ArcGIS Runtime for Android, so both of these issues might be addressed when viewshed is fully supported.

**Comment by conklinbd** [comment]
@garys-esri when Will Crick was demoing the functionality the other day he had similar issues. I think that you are correct that this is a beta issue. I do not think it is a Squad Leader app issue so it is good to defer until the next release.

**Comment by garys-esri** [comment]
I agree. This happened when I was introducing Squad Leader to @KenHarrigan yesterday. I hadn't seen it before, but I think it's rare enough that you don't normally see these behaviors unless you do lots and lots of viewsheds. We'll re-address these issues when viewshed is out of beta.
